packages/forms/src/model.ts

```diff
@@ -999,7 +974,11 @@ export abstract class AbstractControl {
    * * `this.form.get(['items', 0, 'price']);`
    */
   get(path: Array<string|number>|string): AbstractControl|null {
```

Comment:
I guess this should be
```suggestion
  get(path: Array<string|number>|string|null|undefined): AbstractControl|null {
```

Reply:
We do not want to change the existing method signature of `get` -- it should not be necessary to accept `null` or `undefined` as input.

Reply:
Why check for null or undefined then? o: